### ISSUE TITLE
One Piece: Unlimited World Red semi-hangs when starting a new game

### ISSUE THREAD

**Issue by Leo626** [citra-emu/citra#1493]
Using master 3789de6 as of Merge pull request #1476 from lioncash/emit. When I say semi-hangs, the clouds in the background are moving but nothing happens.

![citra-qt_2016-03-10_09-12-59](https://cloud.githubusercontent.com/assets/17358613/13670722/e23777e0-e6a1-11e5-828b-549090c8cea4.png)

[LOG](http://pastebin.com/d4biH3rU)

**Comment by mailwl** [comment]
maybe problem in unimplemented ControlArchive.
May you test my branch? https://github.com/mailwl/citra/tree/ctrl_archive

**Comment by JohnGodgames** [comment]
Tried ctrl_archive and it isn't helped also spamming this:

```
[  74.568686] Service.IR <Warning> core\hle\service\ir\ir.cpp:Service::IR::RequireConnection:61: (STUBBED) called
[  74.569301] Service.IR <Warning> core\hle\service\ir\ir.cpp:Service::IR::FinalizeIrNop:86: (STUBBED) called
[  75.200762] Kernel.SVC <Warning> core\hle\svc.cpp:SVC::CreateMemoryBlock:800: (STUBBED) called addr=0x08001000
[  75.201195] Service.IR <Warning> core\hle\service\ir\ir.cpp:Service::IR::InitializeIrNopShared:51: (STUBBED) called, transfer_buff_size=4096, recv_buff_size=82, unk1=1, send_buff_size=18, unk2=1, baud_rate=4, handle=0x0014811B
[  75.202324] Service.IR <Warning> core\hle\service\ir\ir.cpp:Service::IR::GetConnectionStatusEvent:78: (STUBBED) called
```

**Comment by Leo626** [comment]
It looks like I'm getting something similar to @JohnGodgames. Semi-hangs when starting a new game as usual.

[LOG](http://pastebin.com/EEhQfECr)

**Comment by Leo626** [comment]
Whoops accidentally closed this issue. Looks like I'm getting something similar as well.

[LOG](http://pastebin.com/EEhQfECr)

**Comment by Leo626** [comment]
Fixed with MerryMage's [audio branch](https://github.com/MerryMage/citra/tree/audio).

**Comment by Leo626** [comment]
Fixed by 07411fb.
